geometry/identity_test.cpp

```diff
@@ -71,7 +74,8 @@ TEST_F(IdentityTest, Inverse) {
   Vector<quantities::Length, World2> const vector2 =
       Vector<quantities::Length, World2>(
           R3(1.0 * Metre, 2.0 * Metre, 3.0 * Metre));
-  EXPECT_THAT(Id::Identity().Inverse()(vector2).coordinates(),
+  Id identity;
```

Comment:
Not sure whether another `Id` object is needed with one being created a few lines below, but figured I should stay on the conservative side, since I'm not familiar with the code.